101-bakup-vm-recovery-services-vault/azuredeploy.json

```diff
@@ -0,0 +1,83 @@
+{
+
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+
+  "contentVersion": "1.0.0.0",
+
+  "parameters": {
+
+    "vaultName": {
+
+      "type": "string",
+
+      "defaultValue": "myRSVault"
+
+    },
+
+    "vmResourceGroupName": {
+
+      "type": "string",
+
+      "defaultValue": "myVMRG"
+
+    },
+
+    "vmName": {
+
+      "type": "string",
+
+      "defaultValue": "myVM"
+
+    },
+
+    "policyName": {
+
+      "type": "string",
+
+      "defaultValue": "DefaultPolicy"
+
+    }
+
+  },
+
+  "variables": {
+
+    "protectionContainers": "[concat('iaasvmcontainer;iaasvmcontainerv2;', parameters('vmResourceGroupName'), ';', parameters('vmName'))]",
+
+    "protectedItems": "[concat('vm;iaasvmcontainerv2;', parameters('vmResourceGroupName'), ';', parameters('vmName'))]",
+
+    "protectedItemType": "Microsoft.Compute/virtualMachines",
```

Comment:
Please confirm this, I guess this should be: Microsoft.ClassicCompute/virtualMachines while passing this to "properties". However, for constructing v2 vm's resourceId this is correct.